smdebug/tensorflow/keras.py

```diff
@@ -394,9 +397,9 @@ def save_custom_tensor(self, tensor_name, tensor_value, collections_to_write):
     def _save_custom_tensors_post_step(self):
         for tensor_name in self.custom_tensors_to_save:
             tensor_value, collection_names = self.custom_tensors_to_save[tensor_name]
-            self._save_tensor(tensor_name, tensor_value, collection_names)
+            self._save_tensor_to_file(tensor_name, tensor_value, collection_names)
 
-    def _save_tensor(self, tensor_name, tensor_value, collections_to_write):
```

Comment:
is there a conflict for the name _save_tensor() ? or are you changing this to avoid confusion with hook.save_tensor()?

Reply:
I'm changing this name to avoid confusion